docker-compose.yml

```
@@ -116,6 +121,7 @@ services:
     volumes: *default-volumes
     environment:
       - MAPS_API_V3_RESOURCE_URL_TEMPLATE_NEW=https://$${host}/api/$${path}
+      - NODE_OPTIONS=--max-old-space-size=3072
```

If we stick to the cloud values, it should be 2048

```suggestion
      - NODE_OPTIONS=--max-old-space-size=2048
```